CHANGELOG.md

```diff
@@ -31,6 +31,10 @@
 
 N/A
 
+## Changes since v7.5.0
+
+- [#1922](https://github.com/oauth2-proxy/oauth2-proxy/pull/1922) Added support for env variables in the alpha struct
+
```

Comment:
@hevans-dglcom please move to line 15. (existing `Changes since v7.5.0`) section

Reply:
done